content/docs/concepts/architecture/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Architecture
+description: Learn about the architecture, how Keptn works internally, and can be extended.
+weight: 90
+keywords: [keptn, architecture]
+---
+
+Keptn is an event-based control plane for continuous delivery and automated operations that runs on Kubernetes. When installing Keptn on a Kubernetes cluster, core components, and Istio are installed. Keptn itself follows an event-driven approach with the benefits of loosely coupled components and a flexible design that allows easily integrating other components and services. The events that Keptn understands are specified [here](https://github.com/keptn/spec/blob/0.1.3/cloudevents.md) and follow the [CloudEvents](https://cloudevents.io/) specification [v2](https://github.com/cloudevents/spec/tree/v0.2).
+
+{{< popup_image link="./assets/architecture.png" caption="Keptn architecture">}}
+
+## Prerequisites
+
+During a Keptn installation, the following components are installed on the Kubernetes cluster:
+
+- [Istio](https://istio.io)
+- [NATS](https://nats.io/)  
+
+## Keptn CLI
+The Keptn CLI needs to be installed on the local machine and is used to send commands to Keptn by interacting with the API of Keptn. To communicate with Keptn you need to know a shared secret that is generated during the installation and verified by the *api* component.
+
+**Note:** A dedicated Keptn CLI section is provided [here](../../develop/reference/cli/), which helps you to get started and lists all available commands.
+
+## Keptn core components 
+
+Keptn has several core components that are set up during the installation.
+
+### api
+
+The *api* component provides a publicly exposed REST API that allows communicating with Keptn. It receives the commands from the CLI and executes the requested tasks. However, the api component mostly uses other internal services or forwards events to the *eventbroker*. In addition to the REST API, the api component maintains a websocket server to forward Keptn messages to the *Keptn CLI*.
+
+**Note:** A dedicated Keptn API section is provided [here](../../develop/reference/api/), which helps you to access the API and to explore the available endpoints.
+
+### eventbroker
+
+The *eventbroker* is responsible for receiving all events, transferring non-Keptn events into valid Keptn CloudEvents, and sending those into NATS.
```

Comment:
The eventbroker does not transfer non-Keptn events into valid Keptn CloudEvents. This already takes place in the api.

Reply:
I have removed the wrong info.